xml/System.Windows.Forms/TaskDialogIcon.xml

```diff
@@ -18,8 +18,25 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents an icon that can be shown in the main area of a task dialog (by setting the <see cref="P:System.Windows.Forms.TaskDialogPage.Icon" /> property) or in the footnote of a task dialog (by setting the <see cref="P:System.Windows.Forms.TaskDialogFootnote.Icon" /> property).</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+ The task dialog currently supports two icon types: Standard icons that are stored in static fields of class <xref:System.Windows.Forms.TaskDialogIcon>, and custom icons created from an <xref:System.Drawing.Icon> instance (or an icon handle).
+            
+
+ Some standard icons play a typical system sound when used as the main icon of the task dialog.
+            
+
+ Note that while a task dialog is shown, you can only update an icon if the new icon is of the same type (standard icon or custom icon) as the previous one.
```

Comment:
Convert note

Reply:
I think this one may not have been intended as an actual `> [!NOTE]`.